### DISCUSSION TITLE
天津大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#20] [University]
> 回答来自群友，我仅负责搬运

## 老校区
**宿舍是上床下桌吗**
本科生部分6人间，部分4人间（学院决定），都是上床下桌。

**教室和宿舍有没有空调**
大部分教学楼和宿舍都有。宿舍空调需要租金，和舍友平摊下来不太贵。

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
部分宿舍没有需要去澡堂（100米之内），部分宿舍每层层浴。

**有早自习、晚自习吗**
无。

**有晨跑吗**
有跑步锻炼记录要求，为本科生必修课（0学分）。

**每学期跑步打卡的要求是多少公里，可以骑车吗**
按照教学周安排，每周3次及格4次良好5次优秀，恶劣天气降低要求。不能骑车。男生每次1.5km，女生每次1km。

**寒暑假放多久，每年小学期有多长**
寒假一个半月，暑假也差不多一个半月，小学期一个月。

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许。根据疫情防控的有关要求，目前取外卖是在校门口。

**学校附近有地铁站吗**
附近没有，做高分辨的实验很放心。

**宿舍楼有洗衣机吗**
有。

**校园网怎么样**
千兆到楼，百兆到端。 v4每月10G免费额度，超出部分2元/G。 v6免费。（注：根据疫情防控的有关要求，目前全免费）

**每天断电断网吗，几点开始断**
周一至周五23：39到次日早上6：00左右断电，周末00：39-6：00断电，考试周节假日通宵供电。空调插座不断电，宿舍不断网。

**食堂价格贵吗，会吃出异物吗**
贵，小概率

**洗澡热水供应时间**
24h

**校园内可以骑电瓶车吗，电池在哪能充电**
可以，自己找地方。

## 新校区

**宿舍是上床下桌吗**
4人间，都是上床下桌。

**教室和宿舍有没有空调**
大部分教学楼和宿舍都有。宿舍空调需要租金，和舍友平摊下来不太贵。

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
每层层浴。

**有早自习、晚自习吗**
无。

**有晨跑吗**
有跑步锻炼记录要求，为本科生必修课（0学分）。

**每学期跑步打卡的要求是多少公里，可以骑车吗**
按照教学周安排，每周3次及格4次良好5次优秀，恶劣天气降低要求。不能骑车。男生每次1.5km，女生每次1km。

**寒暑假放多久，每年小学期有多长**
寒假一个半月，暑假也差不多一个半月，小学期一个月。

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许。根据疫情防控的有关要求，目前取外卖是在校门口。微信小程序有送达至校内的外卖平台，实际上只能送到宿舍楼下，遵循应慢尽慢的原则。

**学校附近有地铁站吗**
附近没有，做高分辨的实验很放心。

**宿舍楼有洗衣机吗**
有。

**校园网怎么样**
千兆到楼，百兆到端。 v4每月10G免费额度，超出部分2元/G。 v6免费。（注：根据疫情防控的有关要求，目前全免费）。实际体验很烂。

**每天断电断网吗，几点开始断**
不断电，不断网。

**食堂价格贵吗，会吃出异物吗**
真nm血贵还贼难吃，小概率

**洗澡热水供应时间**
24h

**校园内可以骑电瓶车吗，电池在哪能充电**
可以，宿舍没有充电桩，需要自己找地方。

**Comment by At-East** [comment]
目前新校区已经通了地铁6号线二期（8号线）

**Comment by 0Jason-0** [comment]
“跑步锻炼记录”并非对每个人强制要求。
它有避免办法（比如体测成绩好），也有其他选择。

**Comment by 0Jason-0** [comment]
老校区（卫津路校区）周围有两座地铁站：南丰路站（8号线）、七里台站（7号线、8号线）。8号线可连通两座校区（8号线开通后连接两校区的615路、615路快线很有可能被撤销）。这两座地铁站距校门都有一定距离，若从宿舍、教学楼出发，骑自行车是一个不错的选择。

**Reply by 0Jason-0** [reply]
7号线、8号线均在建。7号线鼓楼站以南有可能于2024年通车。8号线建设尚需时日。
